mediaqueries-5/Overview.bs

```diff
@@ -1730,6 +1730,115 @@ Color Display Quality: the 'color-gamut' feature</h3>
 	''not (color-gamut)''.
 
+<h3 id="in-gamut">
+	Displayable Color: the 'in-gamut' feature</h3>
+
+	<pre class='descdef mq'>
+		Name: in-gamut
+		Value: <<color>>
+		For: @media
+		Type:  discrete
+	</pre>
+
+	The '@media/in-gamut' media feature
+	is used to query whether a given <<color>>
+	lies within the <a>gamut</a> of the output device,
+	and thus,
+	it can be accurately displayed.
+
+	<div class="example">
+		The following media query tests whether primary green,
+		in the <a>display-p3</a> colorspace,
+		is in gamut for the output device.
+
+		<pre class="lang-css">@media (in-gamut:color(display-p3 0 1 0)) { … }</pre>
+	</div>
+
+	To determine whether the color is in gamut,
+	it is first converted to CIE XYZ or CIE Lab,
+	and then converted to the colorspace of the output device.
+
+	If the value for each color component,
+	in the output colorspace,
+	lies in the range [0.0-1.0],
+	the color is in-gamut.
+	Otherwise, (if there are negative components, or components greater than one),
+	it is out of gamut.
+
+	<!--
+		This example was calculated with
+		https://colorjs.io/apps/convert/?color=color(display-p3%200%201%200)&precision=4
+	-->
+
+	<div class="example">
+		For example, if the display is an sRGB screen,
+		the display-p3 color from the previous example
+		has the following components: red -2.9058, green 1.0183, blue -1.0162.
+		It is therefore out of gamut for this screen.
+	</div>
+
+	If the colorspace of the output device is unknown,
+	then for Web compatibility,
+	it should be assumed to be <a>sRGB</a>.
+
+	<!--
+		This example was calculated with
+		https://colorjs.io/apps/convert/?color=color(a98-rgb%200.48%200.86%200.95)&precision=3
+	-->
+	<div class="example">
+		The following media query tests whether a sky blue,
+		in the <a>a98-rgb</a> colorspace,
+		is out of gamut for the output device,
+		and if so, displays an out of gamut warning.
+
+		<pre class="lang-css">@media not (in-gamut:color(a98-rgb 0.48 0.86 0.95)) {
+				p.swatch {border: thin dashed red; }
+			}</pre>
+	</div>
+
+	The media query is not restricted to RGB output devices.
+
+	<!--
+		This example was calculated using the ICCMax reference implementation
+		and two color profiles, one for FOGRA39 and one for FOGRA55.
+		The data files are in the csswg css-color-4/workings directory
+		The input data is a file of Lab values for the first row on a MacBeth color checker chart.
+		The command line is
+		$ IccApplyNamedCmm MacBeth-Lab.txt 3:6 0 Coated_Fogra39L_VIGC_300.icc 41
+		$ IccApplyNamedCmm MacBeth-Lab.txt 3:6 0 2020_13.003_FOGRA55beta_CL_Profile.icc 41
+		run $ IccApplyNamedCmm with no arguments for usage
+		Macbeth-Fogra39.txt
+		Macbeth-7color.txt
+		MacBeth-Lab.txt
+	-->
+
+	<div class="example">
+		This example tests whether a given color,
+		which represents the cyan on a ColorChecker chart
+		and is specified in FOGRA39 CMYK colorspace,
+		is in gamut for the output device.
+
+		<pre class="lang-css">@color-profile --fogra39 {
```

Comment:
There's no need to do this sort of source contortion - you can just line up all your code properly starting on the next line, and Bikeshed'll ensure the output is good.